Server/middleware/cloudinary.js

```diff
@@ -0,0 +1,21 @@
+import cloudinary from "cloudinary";
+import dotenv from "dotenv";
+import url from "url";
+
+function cloudinar(req,res,next){
```

Comment:
Use es6 syntax for functions. Also add spaces between the parameters `(req,res,next)`

Reply:
Okay